### ISSUE TITLE
🐛 Sorting alphabetically doesn't work

### ISSUE THREAD

**Issue by MatMoore** [ministryofjustice/find-moj-data#218]
### Describe the bug.

When sort is set to "ascending" or "descending", the results are not sorted alphabetically.

Note that sort by ascending does sort by the *table name* ascending, but we display the *fully qualified name*, so it looks incorrectly sorted when there are results from different databases or containers.

### To Reproduce

Do a search that returns tables from multiple databases, e.g. https://data-platform-find-moj-data-test.apps.live.cloud-platform.service.justice.gov.uk/search?query=cases&domain=&sort=ascending

### Expected Behaviour

The sort behaviour should match what is rendered on the page.

Ascending/descending should sort by qualified name if possible

If qualified name is not available, only then should we sort by name

Ideally all the sorting should be handled by the API, so it works with the pagination.

Note: there might still be some edge cases here. If qualified name is not defined, is it possible to filter by `COALESCE(qualifiedName, name)`?

### Additional context

We might want to make other changes to this component at the same time as fixing this

E.g.

- The latest design uses a select instead of radio boxes to conserve vertical space ([figma](https://www.figma.com/file/zTOVhHRnWfxRJqH6Gr8cIc/Catalogue-sketches?type=design&node-id=203-2121&mode=design&t=YocOJ80P7hF7KanZ-0))
- We may have further suggestions after our second UR round

**Comment by YvanMOJdigital** [comment]
wait till ur playback

**Comment by YvanMOJdigital** [comment]
Outcome of playback: hide sort functionality until new design agreed

**Comment by teeceeas** [comment]
Agreed that this feature was not required as research shows users more interested in filters
